### PR TITLE
[DOCS-50] Testing 'Expand' code

### PR DESCRIPTION
**Pull request by mjang-cobalt**
This turned out better than I expected. @dannyleong39 I'd love your opinion/approval of this. I'm going to share with Stef as well before merging.

I think this can be an excellent tool to minimize any "walls of text" in our docs, based on Marketing feedback.

You can review the build here: https://deploy-preview-100--cobalt-docs.netlify.app/getting-started/
Screen Shot 2022-03-01 at 12 14 48 PM

### CODE REVIEW

File: layouts/shortcodes/expand.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{{ $_hugo_config := `{ "version": 1 }` }}
+<div class="expand">
+    <div class="expand-label" style="cursor: pointer;" onclick="$h = $(this);$h.next('div').slideToggle(100,function () {$h.children('i').attr('class',function () {return $h.next('div').is(':visible') ? 'fas fa-chevron-down' : 'fas fa-chevron-right';});});">
+        <i style="font-size:x-small;" class="fas fa-chevron-right"></i>
```
Comment (mjang-cobalt) [inline layouts/shortcodes/expand.html:4]:
```suggestion
        <i style="font-size:large;" class="fas fa-chevron-right"></i>
```

File: layouts/shortcodes/expand.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{{ $_hugo_config := `{ "version": 1 }` }}
+<div class="expand">
+    <div class="expand-label" style="cursor: pointer;" onclick="$h = $(this);$h.next('div').slideToggle(100,function () {$h.children('i').attr('class',function () {return $h.next('div').is(':visible') ? 'fas fa-chevron-down' : 'fas fa-chevron-right';});});">
+        <i style="font-size:large;" class="fas fa-chevron-right"></i>
```
Comment (mjang-cobalt) [inline layouts/shortcodes/expand.html:4]:
```suggestion
        <i style="font-size:x-large;" class="fas fa-chevron-right"></i>
```

**Comment by mjang-cobalt** [comment]
@dannyleong39 Per your suggestion in (internal) Slack, I set `color: blue` in [this line](https://github.com/cobalthq/cobalt-product-public-docs/pull/100/files#diff-b3fa3943eb5ca663d2637d7e8711e5fc91a8eadb94efda16bb6017af954a508bR4), and I think it works nicely. I'm happy to defer to your judgement.

Screen Shot 2022-03-03 at 1 34 43 PM

**Comment by mjang-cobalt** [comment]
Here's a screenshot with the color updated to #0047AB in [this line](https://github.com/cobalthq/cobalt-product-public-docs/pull/100/files#diff-b3fa3943eb5ca663d2637d7e8711e5fc91a8eadb94efda16bb6017af954a508bR4)
Screen Shot 2022-03-09 at 1 50 13 PM

**Comment by mjang-cobalt** [comment]
Thanks for your help, @dannyleong39 I think it's ready for your approval now.

**Review by dannyleong39 — APPROVED** [review]
Thanks, Mike for updating the color. Looks much better!

**Pull request merged**